packages/tokens/dist/products/css/themed-tokens/with-css-selectors--advanced/tokens.css

```diff
@@ -0,0 +1,538 @@
+/**
+ * Do not edit directly, this file was auto-generated.
+ */
+
+:root,
+.hds-theme-default {
+  --token-color-palette-neutral-0: #ffffff;
```

Comment:
Just curious, is there a reason to not use shorthand for the hex color values like this? (i.e., use "#fff" vs. "#ffffff")